src/main/java/seedu/us/among/ui/ResultDisplay.java

```diff
@@ -204,4 +219,37 @@ public void setApiFeedbackToUser(String feedbackToUser, Endpoint endpoint) {
             apiFeedbackHelper(textFeedback, endpoint);
         }
     }
+
+    /**
+     * Sets the flow pane for a unique endpoint. Contains the fields of methods,
+     * status code + phrase, response time and protocol version.
+     * @param endpoint endpoint to give feedback for
+     */
+    public void setResponseMetaToUser(Endpoint endpoint) {
```

Comment:
good job moving the code from apiFeedbackHelper to this function as it improves readability by alot!